src/main/resources/rankups.yml

```diff
@@ -5,40 +5,50 @@
 # Join the discord server for live support: https://discord.gg/maB4382 (buyers only)
 #

```

Comment:
add https://github.com/okx-code/Rankup3/wiki/How-to-rankups.yml-and-prestiges.yml#how-to-rankups

Reply:
i'll wait for the new wiki to be live before adding a link